### ISSUE TITLE
add support for mybook.refresh_all()

### ISSUE THREAD

**Issue by fzumstein** [xlwings/xlwings#1592]
https://docs.microsoft.com/en-us/office/vba/api/excel.workbook.refreshall

**Comment by chfiii** [comment]
i think that given an open workbook 'wbk', this can be done via wbk.api.RefreshAll.  Is this true>

**Comment by fzumstein** [comment]
correct. this issue exists to make it available without the `api` part.

**Comment by chfiii** [comment]
Thanks.  One quit question, if I open the app with visible=False, when I close it (app.close() or app.kill()) maybe with an wbk.close() before the app.quit(), will everything happily shutdown?  I’m using Excel Stock Data Types for getting stock price updates and am afraid of an interaction.

**Comment by chfiii** [comment]
Never mind my question.  I just tested it and it works fine.

Thanks again!

Charlie

**Comment by chfiii** [comment]
Thanks a lot!  I’ll do the separate issue in the future.

**Comment by chfiii** [comment]
Don’t have your personal email but want to say that your ‘Python for Excel’ is a great book!  Extremely helpful for getting into the more esoteric stuff.

**Comment by carecodeconnect** [comment]
Hi @fzumstein thanks for your great package!

I'm updating a legacy codebase running in Windows so that it runs in MacOS. 

Is there a workaround for `wbk.api.RefreshAll()` in MacOS? Either in `xlwings` or an alternative package?

Thanks!

**Comment by fzumstein** [comment]
can you try with `wbk.api.refresh_all()`?
